src/ovs/aca_ovs_l2_programmer.cpp

```diff
@@ -235,7 +194,11 @@ int ACA_OVS_L2_Programmer::create_port(const string vpc_id, const string port_na
             cmd_string, culminative_time);
     if (command_rc != EXIT_SUCCESS)
       overall_rc = command_rc;
-  } else {
+  }
+  /*
```

Comment:
Please uncomment line 198 and line 224, so that it calls `aca_set_port_vlan_workitem` in other test scenarios.

Reply:
Fixed